src/__tests__/register-test.js

```diff
@@ -462,33 +632,105 @@ describe('ethrResolver', () => {
           return expect(resolve(did)).resolves.toEqual({
             '@context': 'https://w3id.org/did/v1',
             id: did,
-            publicKey: [{
-              id: `${did}#owner`,
-              type: 'Secp256k1VerificationKey2018',
-              owner: did,
-              ethereumAddress: owner
-            }, {
-              id: `${did}#delegate-1`,
-              type: 'Secp256k1VerificationKey2018',
-              owner: did,
-              ethereumAddress: delegate2
-            }],
-            authentication: [{
-              type: 'Secp256k1SignatureAuthentication2018',
-              publicKey: `${did}#owner`
-            }, {
-              type: 'Secp256k1SignatureAuthentication2018',
-              publicKey: `${did}#delegate-1`
-            }]
+            publicKey: [
+              {
+                id: `${did}#owner`,
+                type: 'Secp256k1VerificationKey2018',
+                owner: did,
+                ethereumAddress: owner,
+              },
+              {
+                id: `${did}#delegate-1`,
+                type: 'Secp256k1VerificationKey2018',
+                owner: did,
+                ethereumAddress: delegate2,
+              },
+            ],
+            authentication: [
+              {
+                type: 'Secp256k1SignatureAuthentication2018',
+                publicKey: `${did}#owner`,
+              },
+              {
+                type: 'Secp256k1SignatureAuthentication2018',
+                publicKey: `${did}#delegate-1`,
+              },
+            ],
           })
         })
       })
     })
   })
 
+  describe('multiple events in one block', () => {
+    beforeAll(async () => {
+      await stopMining()
```

Comment:
Anyone know of a better way to force 2 transactions to be mined into a single block using ganache?